src/components/activity/ActivityRegistrator.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { View, ScrollView, KeyboardAvoidingView, TouchableWithoutFeedback, Keyboard, Dimensions, Platform  } from 'react-native';
+import { Text, IconButton, Avatar, useTheme } from 'react-native-paper';
+
+import Slider from '@react-native-community/slider';
+
+import { DatePicker } from '../DatePicker';
+import { TimePicker, getCurrentTimeRounded } from '../TimePicker';
+import { SuggestiveTextInput } from '../SuggestiveTextInput';
+
+import { useTranslation } from 'language/LanguageProvider';
+import Storage from 'storage';
+
+export const ActivityRegistrator = ({ route, navigation }: any) => {
+  // route.params contains information from activity screen
+
+  const lang = useTranslation();
+  const { iconSizes, colors } = useTheme();
+
+  const steps = 1;
+  const defaultChoise = {
+    value: lang.activityRegistratorActivityDefaultChoise,
+    isDefault: true
+  }
+  let choises = [ defaultChoise ];
```

Comment:
```suggestion
  let choices = [ defaultChoice ];
```
Make sure all of these are renamed